scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/syntax/SCollectionSyntax.scala

```diff
@@ -54,15 +55,52 @@ final class PredictSCollectionOps[T: ClassTag](private val self: SCollection[T])
    */
   def predict[V: Coder, W](
     savedModelUri: String,
-    fetchOps: Seq[String],
-    options: TensorFlowModel.Options
+    options: TensorFlowModel.Options,
+    fetchOps: Option[Seq[String]] = None,
```

Comment:
Note: API breaking change

Reply:
An alternative would be to still accept a seq from callers, and check if it's empty to return all ops; but I think that would be a strange behavior

Reply:
Thinking about the issue with memory due to multiple instances on the same JVM... won't there one instance + clones made for each machine? If so, than this should be ok because only one model is loaded per machine-instance. I'm not very familiar with how that works